.github/workflows/ci-multiarch.yaml

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+name: CI:multiarch
+
+on:
+  workflow_dispatch:
+    inputs: {}
+  # push:
+  pull_request:
```

Comment:
it's going to be push again before merging, right?

Reply:
correct